src/pages/card.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./card.css";
+import { SlLogin } from "react-icons/sl";
+
+function Card() {
```

Comment:
I tried to run in locally and notice some inconsistencies
<img width="288" alt="image" src="https://github.com/daniel-august-mentorship/Demo-School/assets/6497134/fe307116-7c13-423a-aaaf-3b0abe50d96f">

* the icon should be `right aligned`
* the `150` should be centered in the middle
* the `Since last month` text should be `left aligned` and next to the 20%
* remove the gray background color